src/open_inwoner/openklant/services.py

```diff
@@ -771,17 +763,22 @@ def retrieve_question(
                     "Unable to find matched contactmomenten zaak in any zgw backend"
                 )
             else:
```

Comment:
```suggestion
            elif case_count > 1
```
The two conditions belong to the same logical level

Reply:
But if `case_count != 0` I always want to run the iterator to fetch the first (including `case_count == 1`), but only log the error if `case_count > 1`, so the `else` seemed to make sense to me. It would then have to be `elif case_count >= 1` and I'd _still_ need the extra conditional. The `else` seemed cleaner. Or am I missing something?

Reply:
You're right, this makes sense.